src/test/suites/interface-suite.js

```diff
@@ -186,17 +186,33 @@ suite.declare((lab, variables) => {
       return done();
     });
 
-    lab.test("A null source field throws an error", done => {
+    lab.test("A null source field should map the source to destination", done => {
 
       const map = createSut();
 
-      const throws = function () {
+      const result = map(null).execute({});
 
-        map(null).to("field.name");
+      expect(result).to.equal({});
 
-      };
+      return done();
 
-      expect(throws).to.throw();
+    });
+
+    lab.test("A null source field should map the source to destination provided", done => {
```

Comment:
The name of this test isn't very descriptive. Maybe *A null source field can be used alongside a normal mapping*?